packages/contracts/src/contract-data.ts

```diff
@@ -0,0 +1,174 @@
+import * as path from 'path'
+import { predeploys as l2Addresses } from './predeploys'
+import { Network } from './connect-contracts'
+
+const Mainnet__Lib_AddressManager = require('../deployments/mainnet/Lib_AddressManager.json')
```

Comment:
There is a lot of backstory with this file. Liam requested the same (do the imports with a helper function) but after a ton of struggling, we realized we can't use Node.js stuff (`path`, `fs`, etc) because this is for the browser.

Reply:
One option for a helper function here is something along the lines of:

```ts
const loadContractData = (network: string, name: string) => {
  return require(`../deployments/${network}/${name}.json`)
}
```

Then you could update `getL1ContractData` as follows:

```
export const getL1ContractData = (network: Network) => {
  return {
    Lib_AddressManager: loadContractData(network, 'Lib_AddressManager'),
    ...
  }
}
```

Reply:
The problem is all the json needs to be imported when the typescript gets compiled, so I don't think dynamic imports are possible. The browser can't load json into javascript during runtime AFAIK.

Reply:
Hmmmm I see, give me a sec to think about this...

Reply:
OK for now this is fine, but I'll just request that we add a comment explaining why the code looks the way it looks (or others might have the same reaction that I did).

Reply:
In the future I think we should consider dynamically generating this file to reduce the amount of code floating around.